xbmc/addons/gui/GUIDialogAddonInfo.h

```diff
@@ -27,6 +28,25 @@ enum class PerformButtonFocus
   NO,
 };
 
+enum class EntryPoint
+{
+  INSTALL,
+  UPDATE,
+  SHOW_DEPENDENCIES,
+};
+
+struct CInstalledWithAvailable
```

Comment:
It's a struct so `SInstalledWithAvailable`

Reply:
we have `struct CAddonWithUpdate;` but also lots of unprefixed structs across the base.. gotten inconsistent over time.

Reply:
No worries, just a comment.